README.md

```diff
@@ -122,4 +122,6 @@ Usage of /go/src/gotestwaf/gotestwaf:
         If true, enable verbose logging (default true)
   --workers int
         The number of workers to scan (default 200)
+  --ws string
+        WS URL to check
```

Comment:
What is WS? workstation, websocket, Wall Street?

Reply:
Definitely Wall Street! 💰 
![Fxd](https://media.giphy.com/media/gTURHJs4e2Ies/giphy.gif)

Fixed, of course. Also in README too. 
```
  --wsURL string
        WebSocket URL to check
```